charts/workflow/values.yaml

```diff
@@ -107,6 +113,10 @@ controller:
   # disabled - turns off open registration
   # admin_only - allows for registration by an admin only.
   registration_mode: "admin_only"
+  # The public resolvable hostname to build your cluster with.
```

Comment:
Even with the following line clarifying it, I somehow still feel thrown off by the phrase "to build your cluster with."

Can we maybe replace this and the following sentence with something like:

```
A resolvable domain or subdomain name. All applications created by the controller will, by default, be subdomains of this. When experimental native ingress is enabled, these subdomains are used in defining host-based routing rules. If, for instance, platform_domain were "example.com", for the application "foobar", the controller will create rules to route `foo.example.com` to the appropriate backend.
```

^ And that's totally imperfect. Feel free to iterate on that if you can improve it further.